internal/commands/convert.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/plexsystems/konstraint/internal/rego"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newConvertCommand() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "convert <dir>",
+		Short: "Convert old style annotations to OPA Rich Metadata Annotations",
+		Example: `Convert all found policies inplace
```

Comment:
Consider `Convert all policies with legacy annotations to OPA Metadata annotations in-place` instead for clarity.